nifi-commons/nifi-sensitive-property-provider/src/main/java/org/apache/nifi/properties/StandardSensitivePropertyProviderFactory.java

```diff
@@ -74,6 +77,18 @@ private StandardSensitivePropertyProviderFactory(final String keyHex, final Supp
         this.keyHex = Optional.ofNullable(keyHex);
         this.bootstrapPropertiesSupplier = bootstrapPropertiesSupplier == null ? () -> null : bootstrapPropertiesSupplier;
         this.providerMap = new HashMap<>();
+        this.customPropertyContextLocationMap = null;
+    }
+
+    private void populateCustomPropertyContextLocationMap() {
+        final BootstrapProperties bootstrapProperties = getBootstrapProperties();
+        customPropertyContextLocationMap = new HashMap<>();
+        final String xmlContextLocationMappingKeyPrefix = BootstrapPropertyKey.CONTEXT_LOCATION_MAPPING.getKey();
```

Comment:
Recommend renaming the variable to avoid the XML reference.
```suggestion
        final String contextMappingPrefixKey = BootstrapPropertyKey.CONTEXT_LOCATION_MAPPING.getKey();
```